bib/cmd/bootc-image-builder/main.go

```diff
@@ -171,7 +171,16 @@ func saveManifest(ms manifest.OSBuildManifest, fpath string) error {
 	return nil
 }
 
-func manifestFromCobra(cmd *cobra.Command, args []string) ([]byte, *mTLSConfig, error) {
+func manifestFromCobra(cmd *cobra.Command, args []string, pbar progress.ProgressBar) ([]byte, *mTLSConfig, error) {
+	if pbar == nil {
```

Comment:
I'm thinking about `progress.ProgressBar` as a resource, because it needs to be `Stop`ped at some point. So I think there should be a clean owner that's responsible for calling the `Stop` method. This logic currently makes the ownership murky. Can we restructure it somehow?